### ISSUE TITLE
bbox shifted a bit in the predictions using pretrained weights

### ISSUE THREAD

**Issue by amardeepjaiman** [CJHMPower/Simultaneous-Traffic-Sign-Detection-and-Classification-with-RetinaNet#4]
I used the pretraied weights provided in the repo to generate some predictions on the existing sample images and few other sample from google images. But I am getting the bbox location a bit shifted but in the author's sample prediction images those bbox are not shifted. I am attaching few images here. What could be the reason of this as I am just using the pre-trained weights and running the test.py script.

![result394_2_1 jpg](https://user-images.githubusercontent.com/7931015/50132790-d6092780-02ae-11e9-9b34-3691e68e9234.jpg)
![result1642_2_1 jpg](https://user-images.githubusercontent.com/7931015/50132791-d6092780-02ae-11e9-8caf-6189f8525183.jpg)
![result4918_1_1 jpg](https://user-images.githubusercontent.com/7931015/50132792-d6092780-02ae-11e9-88bc-df4ddb15c4f6.jpg)
![result16065_1_1 jpg](https://user-images.githubusercontent.com/7931015/50132793-d6092780-02ae-11e9-8158-2889e87285f4.jpg)
![result39375_0_1 jpg](https://user-images.githubusercontent.com/7931015/50132794-d6a1be00-02ae-11e9-8b13-a4e3fd7a3fd1.jpg)
![resultpedestrian_010 png](https://user-images.githubusercontent.com/7931015/50132795-d6a1be00-02ae-11e9-8340-3c400f01f93f.jpg)
![resultt2 jpg](https://user-images.githubusercontent.com/7931015/50132796-d6a1be00-02ae-11e9-8acc-3ada5e054578.jpg)
![resultt3 jpg](https://user-images.githubusercontent.com/7931015/50132797-d73a5480-02ae-11e9-852e-70e987cecd37.jpg)
![resultt4 jpg](https://user-images.githubusercontent.com/7931015/50132798-d73a5480-02ae-11e9-9c8e-afd362cafed4.jpg)
![resultt5 jpg](https://user-images.githubusercontent.com/7931015/50132799-d73a5480-02ae-11e9-8635-17364b863631.jpg)
![resultwgfuv png](https://user-images.githubusercontent.com/7931015/50132800-d7d2eb00-02ae-11e9-8455-6529624112c2.jpg)

**Comment by amardeepjaiman** [comment]
@CJHMPower , kindly suggest.

**Comment by CJHMPower** [comment]
Hi，I checked the accuracy predicted by the pretrained weight I provided and it works well. I think your problem may be caused by your visualization code. You can directly get the bbox results of my model in the "data" folder(resnet101_predict.json). You can refer this [code file](https://github.com/CJHMPower/Simultaneous-Traffic-Sign-Detection-and-Classification-with-RetinaNet/blob/master/evaluate/eval_check.ipynb) for visualizing the bounding box. Good luck~

**Comment by amardeepjaiman** [comment]
I ran following command : 
python test.py -m demo

and save the predictions instead of plotting them.

**Comment by xavialex** [comment]
Hi all.

First, thanks for the project, it's being really helpful and interesting. I can confirm that I've the same problems in the detection coordinates. I've found out that is because of the PyTorch version. From version 1.0.0 downgraded to 0.4.0 the detection bounding boxes appear where they should again. I'd like to alter the code to make it compatible, but I don't know where the problem might be, besides a warning fixed by changing *F.upsample* by *F.interpolate* in *retinanet.py*.

**Comment by ziqi-zhang** [comment]
My environment is 0.4.1 and I encountered the same problem.

**Comment by wangzhengboo** [comment]
I have tried pytorch1.0,0.3.0 and 0.4.0. And I encountered the same problem.
